official/vision/beta/projects/yolo/utils/iou_ops.py

```diff
@@ -0,0 +1,116 @@
+"""Intersection over union calculation utils."""
+
+# Import libraries
```

Comment:
Remove comment

Reply:
Fixed in https://github.com/tensorflow/models/commit/f28269c8be3c616251af66317f53bf4a4f75f187